plugins/download.py

```diff
@@ -69,6 +69,8 @@ def configure(self, args):
                                  help=_('packages to download'))
         self.parser.add_argument("--source", action='store_true',
                                  help=_('download the src.rpm instead'))
+        self.parser.add_argument("--debuginfo", action='store_true',
+                                 help=_('download the -debuginfo package instead'))
```

Comment:
can add `--source` and `--debuginfo` options to the same group to be mutually exclusive. please? see repoquery.py how it's done